src/storage/src/hummock/backup_reader.rs

```diff
@@ -184,6 +214,31 @@ impl BackupReader {
         self.inflight_request.lock().remove(&snapshot_id);
         result
     }
+
+    pub async fn watch_config_change(
+        &self,
+        mut rx: tokio::sync::watch::Receiver<SystemParamsReaderRef>,
+    ) {
+        loop {
+            if rx.changed().await.is_err() {
+                break;
+            }
+            let p = (*rx.borrow()).load();
+            let config = (
+                p.backup_storage_url().to_string(),
+                p.backup_storage_directory().to_string(),
+            );
```

Comment:
`rx.changed().await` will return when **any** system param changes, so we need to check if these two params differ from the old ones. 

Or do you think we should have a separate channel for each parameter, to make the interface more friendly?